### DISCUSSION TITLE
French inclusive translation?

### DISCUSSION THREAD

**Discussion by mcanouil** [quarto-dev/quarto-cli#1309] [Q&A] [answered]
Hi,

I wonder if the translation should be inclusive, for example: "Auteur" would be "Auteur·e" and "Auteurs" would be "Auteur·e·s".

https://github.com/quarto-dev/quarto-cli/blob/291055542544e0a1c93747fd9a00e1621ac6fa3a/src/resources/language/_language-fr.yml#L14-L15

Happy to make a PR if deemed necessary.

**Comment by maelle** [comment]
pkgdown uses such vocabulary for French https://github.com/r-lib/pkgdown/blob/main/po/R-fr.po

Note that "Auteur" is "Auteur·rice" in that case (I know both forms exist, _I_ prefer the second one)

**Comment by cscheid** [comment, marked as answer]
Yes, please do make a PR, and thank yoU!

**Reply by mcanouil** [reply]
Done in #1314
